timbermill-java/timbermill-local/pom.xml

```diff
@@ -102,6 +102,13 @@
             <artifactId>kamon-bundle_2.11</artifactId>
             <version>${kamon.bundle.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>io.kamon</groupId>
+            <artifactId>kamon-prometheus_2.11</artifactId>
```

Comment:
I think in dato we use newer version 2.13, if you want to be aligned for 3PP effort

Reply:
Good question, the 2.11 -> 2.13 is related to scala version. Don't think it matters in terms of vulnerabilities. Need to check tho